src/components/sections/judge/JudgeDeduplicatedFinding.tsx

```diff
@@ -0,0 +1,318 @@
+                    </div>
+
+                    <div className="flex w-full justify-end">
+                      <Button type="submit">Save</Button>
+                    </div>
+                  </Form>
+                </DialogContent>
+              </DialogRoot>
+            </div>
+            <div>
+              <FindingSeverityBadge severity={deduplicatedFinding.severity} />
+            </div>
+            <span className="text-titleS">
+              {deduplicatedFinding.contest.title}
+            </span>
+            <p>{deduplicatedFinding.description}</p>
+          </div>
+        </div>
+      </div>
+      <div className="mt-12">
+        <Tabs
+          value={tab}
+          onValueChange={setTabValue}
+          className="flex flex-grow flex-col">
+          <TabsList className="self-start px-24">
+            <TabsTrigger value={JudgeDeduplicatedFindingTab.REPORTS}>
+              reports
+            </TabsTrigger>
+            <TabsTrigger value={JudgeDeduplicatedFindingTab.ADD_FINDINGS}>
+              Add findings
+            </TabsTrigger>
+          </TabsList>
+
+          <Separator />
+          <div className="flex flex-grow flex-col bg-black px-24 pb-24 pt-12">
+            <TabsContent
+              value={JudgeDeduplicatedFindingTab.REPORTS}
+              className="flex flex-col gap-4">
+              {findings.map((finding) => (
+                <div
+                  key={finding.id}
+                  className="flex h-[72px] w-full items-center justify-between bg-grey-90 p-4">
+                  <Button asChild variant="link">
+                    <Link
+                      href={PATHS.judgeFinding(finding.id)}
+                      className="flex gap-6">
+                      {deduplicatedFinding.bestFindingId === finding.id && (
+                        <div className="flex items-center gap-1">
+                          <VerifiedIcon width={24} height={24} />
+                          <span className="text-bodyS">Best</span>
+                        </div>
+                      )}
+                      <span className="text-titleS">{finding.title}</span>
+                    </Link>
+                  </Button>
+                  <div className="flex items-center gap-6">
+                    <div>
+                      <span className="text-bodyM opacity-70">
+                        Submited by:{' '}
+                      </span>
+                      <span className="text-bodyM">
+                        {finding.author.alias ?? finding.author.id}
+                      </span>
+                    </div>
+                    <Button
+                      className="flex items-center gap-2"
+                      size="small"
+                      variant="outline"
+                      onClick={() => removeDeduplicatedFinding(finding.id)}>
+                      <span>Remove</span>
+                      <Trash2 width={16} height={16} />
+                    </Button>
```

Comment:
Don't show the remove button, when it's not possible to remove the finding (if it's the best one).